src/osyris/plot/render.py

```diff
@@ -30,13 +30,24 @@ def render(x=None, y=None, data=None, logx=False, logy=False, ax=None):
         "vec": "quiver",
         "vector": "quiver",
         "stream": "streamplot",
+        "lic": "line_integral_convolution",
         None: config.parameters["render_mode"],
         "image": "pcolormesh",
         "imshow": "pcolormesh"
     }
 
+    for item in data: #check if user requested a line integral conv
```

Comment:
I think this mechanism is a little strange.

To be more in-line with how the other layer modes work, I think the `lic` mode should just behave like a vec mode, where the values for the colormap would be an additional argument, e.g. `color`.

Basically, you would make a `lic` plot by doing
```Py
osyris2.plane(
            {"data": data["hydro"]["velocity"], "mode": "lic", "color": data["hydro"]["density"], "norm": "log"},
            dx=dx * osyris2.units("au"),
            origin=center,
            direction="z",
            resolution=512)
```

I think this means that this code block would go away, and the `line_integral_convolution` wrapper would call the `pcolormesh` wrapper if the `color` is specified, and then fiddle with the colors by adding the `lic` component on top.

(Note that using the `color` argument is how you would set colors of the arrows on a quiver mode.)

Reply:
You're right, that makes more sense. I'll try implementing it.